cpp/examples/abm_minimal.cpp

```diff
@@ -165,10 +147,20 @@ int main()
     mio::abm::close_social_events(t_lockdown, 0.9, world.parameters);
 
     auto t0   = mio::abm::TimePoint(0);
-    auto tmax = mio::abm::TimePoint(0) + mio::abm::days(30);
+    auto tmax = mio::abm::TimePoint(0) + mio::abm::days(10);
```

Comment:
```suggestion
    auto tmax = t0 + mio::abm::days(10);
```
A minor change, but maybe it makes the intention clearer?